content/projects/gsoc/2022/projects/automatic-git-cache-maintenance.adoc

```diff
@@ -63,6 +64,11 @@ image:/images/post-images/gsoc-git-cache-maintenance/maintenance-records.png[tit
 * link:https://github.com/jenkinsci/git-client-plugin/releases/tag/git-client-3.12.0[Changelog]
 * link:https://github.com/jenkinsci/git-client-plugin/blob/git-client-3.4.0/README.adoc[Documentation]
 
+=== Resources
+1. Phase 2 demo and presentation:
+
+video::fM2SMbppRxw[youtube,start=1791,width=800,height=420]
+
 === Outcomes
 
 I cloned the linux kernel and scheduled git maintenance tasks. Below is the analysis of the git maintenance tasks. The repo was 813 commits behind the latest commit while performing the maintenance tasks.
```

Comment:
```suggestion
I cloned the linux kernel and scheduled git maintenance tasks.
Below is the analysis of the git maintenance tasks.
The repo was 813 commits behind the latest commit while performing the maintenance tasks.
```
a small suggestion to make each sentence its own line, otherwise everything looks great, thanks!